src/Http/Requests/StaticPageCreateRequest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Ourgarage\StaticPages\Http\Requests;
+
+use Illuminate\Contracts\Validation\Validator;
+use Notifications;
+use App\Http\Requests\Request;
+
+class StaticPageCreateRequest extends Request
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        $rules = [
+            'title' => 'required',
+            'content' => 'required',
+            'slug' => 'required',
+            'meta_keywords' => 'required',
+            'meta_description' => 'required',
```

Comment:
Meta-tags are not required.
We have to have defaults, in case they are not set.